completions/cmux.bash

```
@@ -0,0 +1,305 @@
+# cmux shell completions (bash) -- AUTO-GENERATED, DO NOT EDIT.
+#
+# Regenerate with:  scripts/generate-cli-completions.py --write
+# Source of truth:  topLevelCommandNames in CLI/cmux.swift + its usage() help.
+
+_cmux() {
+    local cur prev words cword
```

Shellcheck (0.11.0)

[warning] words appears unused. Verify use (or export if used externally).

(SC2034)